src/App.tsx

```diff
@@ -1,60 +1,133 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [hasUserError, setHasUserError] = useState(false);
+  const [titleErrorMessage, setHasTitleErrorMessage] = useState('');
+  const hasTitleError = titleErrorMessage !== '';
+
+  function getNewPostId(todosList: Todo[]) {
+    const maxId = Math.max(...todosList.map(todo => todo.id));
+
+    return maxId + 1;
+  }
+
+  const isValidTitle = (inputedValue: string) => {
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. Consider renaming 'inputedValue' to a more descriptive name that follows the checklist item 1.